src/Microsoft.AspNet.Mvc/MvcOptionsSetup.cs

```diff
@@ -59,6 +60,10 @@ public static void ConfigureMvcOptions(MvcOptions options)
             options.ValueProviderFactories.Add(new QueryStringValueProviderFactory());
             options.ValueProviderFactories.Add(new FormValueProviderFactory());
 
+            // Set up metadata providers
+            options.ModelMetadataProviders.Add(new DefaultBindingMetadataProvider());
+            options.ModelMetadataProviders.Add(new DataAnnotationsMetadataDetailsProvider());
```

Comment:
suggestions?

DefaultBindingMetadataProvider is-a IBindingMetadataProvider
DataAnnotationsMetadataDetailsProvider is-a IBindingMetadataProvider and IDisplayMetadataProvider